### DISCUSSION TITLE
How to turn an animation with 'loop: true' on and off

### DISCUSSION THREAD

**Discussion by baesumin** [nandorojo/moti#299] [Q&A]
How to turn an animation with 'loop: true' on and off?

<MotiView
        from={{
          rotation: isEditMode ? -2 : 0,
        }}
        animate={{
          rotation: isEditMode ? 2 : 0,
        }}
        transition={{
          loop: true,
          type: 'timing',
          duration: 200,
        }}>

 I want to stop the loop in a specific situation and start it again.

**Comment by nandorojo** [comment]
i don’t know if this works, but did you try setting it to false when the state updates?
